skywalker/widgetgroup.py

```diff
@@ -356,17 +356,13 @@ def setup(self, *, pvnames, name=None, rotation=0, **kwargs):
             width_pv = pvnames[0]
             image_pv = pvnames[1]
             image_item = img_widget.getImageItem()
-            image_item.setTransformOriginPoint(self.raw_size_x//2,
-                                               self.raw_size_y//2)
-            image_item.setRotation(rotation)
+            image_item.setTransformOriginPoint(self.size_x/2,
+                                               self.size_y/2)
+            image_item.setRotation((rotation + 90) % 360)
```

Comment:
Are the other clients of those cameras doing the rotation by code?
Wouldn't it be better to agree to modify it on the Area Detector side?
I don't mind the current implementation... I'm just concerned for the problems that different implementations can cause.

Reply:
Yes, the camviewer is doing the rotation by code. I think the problem here is that doing a rotation in AreaDetector maxes out the cpu load on these nodes.